testsuite/features/build_validation/init_clients/centos8_minion.feature

```diff
@@ -4,29 +4,29 @@
 #  1) bootstrap a new CentOS 8 minion via salt-ssh
```

Comment:
This is not via salt-ssh, just normal salt-minion, please change this as well.

Reply:
Correct.